docs/INSTALL.md

```diff
@@ -130,7 +130,7 @@ The following example uses Bash (default) on macOS (Intel).
 3. Extract the plugin.
 
     ```console
-    tar xvf terraform-provider-vsphere_${RELEASE}_darwin_amd64.zip
+    gunzip terraform-provider-vsphere_${RELEASE}_darwin_amd64.zip
```

Comment:
Why not just `unzip` and provide the destination path?

```console
unzip terraform-provider-vsphere_${RELEASE}_darwin_amd64.zip -d terraform-provider-vsphere_${RELEASE}
```

Then

```console
mv terraform-provider-vsphere_v${RELEASE}/terraform-provider-vsphere_v${RELEASE} ~/.terraform.d/plugins/local/hashicorp/vsphere/${RELEASE}/darwin_amd64
```

Reply:
yes sure this is a much better approach

Reply:
Do you want to update your commit?

Reply:
Addressed in force push of 8e47f6f .